module/VuFind/src/VuFind/Config/Feature/ConfigSettingPropertiesTrait.php

```diff
@@ -0,0 +1,252 @@
+<?php
+
+/**
+ * Trait implementing SettingPropertiesInterface
```

Comment:
Resynchronizing name:

```suggestion
 * Trait implementing ConfigSettingPropertiesInterface
```